lightcycle-integration-test/src/main/java/com/soundcloud/lightcycle/sample/basic/ActivityLogger.java

```diff
@@ -0,0 +1,63 @@
+package com.soundcloud.lightcycle.sample.basic;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.MenuItem;
+
+import com.soundcloud.lightcycle.sample.basic.callback.ActivityCallback;
+import com.soundcloud.lightcycle.sample.basic.callback.ActivityLifecycleCallback;
+
+public class ActivityLogger extends BaseActivityLogger<SampleActivity> {
```

Comment:
This is illegal `ActivityLogger` cannot dispatch `SampleActivity` because `ActivityLogger` does not extend `SampleActivity`. Actually, this exposes an issue : it should not compile!